app/posts/teacher-success/2025-04-25-testing-support-comms-with-trainees.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Testing supportive comms with trainee teachers
+description: Detailing email comms we sent to trainees to test their engagement with supportive content from Department for Education
+date: 2025-04-25
+tags:
+  - withdrawals
+  - trainees
+  - emails
+  - comms
+---
+
+We wanted to test engagement from trainees with supportive email comms from Department for Education.
+
+[View the full comms testing report](https://educationgovuk.sharepoint.com/:p:/r/sites/TeacherServices/Shared%20Documents/TWD%20-%20Teaching%20Vacancies/TEACHER%20SUCCESS%20SERVICE%20ALPHA%20-%202025/Design%20and%20content/Comms%20testing%20results.pptx?d=w181456ee8022433ab92d68bd2cb6c458&csf=1&web=1&e=35y4Ln).
+
+## Method
+
+Using the 2024/25 trainee data from Register trainee teachers, we used the Mailchimp platform to email all current trainees.
+
+## Testing limitations
+
+At this point in the cycle (March/April), most training is already well under way. This
+will likely have an impact on engagement if the need for this information is up front,
+in which case, we've missed the boat.​
+
+We're mainly signposting to existing content provided externally – we will not be able
+to measure how useful the content is or what issues they're having that led them to
+click on support links.
+
+## What we tested
+
+We used a multivariate test with 4 variations:
+
+- non-teaching branded, lesson planning then get support
+- non-teaching branded, get support then lesson planning​
+- teaching branded, lesson planning then get support​
+- teaching branded, get support then lesson planning
+
+We wanted to make sure that the ordering of the content did not have an impact on what users clicked on so we sent different versions of the email where we changed the order.
+
+We also wanted to test engagement with a teaching branded email vs a more GDS style email.
```

Comment:
```suggestion
We also wanted to test engagement with a Get Into Teaching-branded email versus a more GOV.UK style email.
```

Reply:
Whilst the style guide and Notify are products/services/guidance managed by GDS, they are more appropriately referred to as "GOV.UK" (i.e., GOV.UK style guide, GOV.UK Notify, and so on).